app/resources/numismatic_accessions/numismatic_accession_change_set.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+class NumismaticAccessionChangeSet < Valkyrie::ChangeSet
+  delegate :human_readable_type, to: :model
+
+  property :date, multiple: false, required: false
+  property :person, multiple: false, required: false
+  property :firm, multiple: false, required: false
+  property :accession_number, multiple: false, required: false
+  property :type, multiple: false, required: false
+  property :cost, multiple: false, required: false
+  property :account, multiple: false, required: false
+  property :note, multiple: false, required: false
+  property :private_note, multiple: false, required: false
+
+  property :member_ids, multiple: true, required: false, type: Types::Strict::Array.of(Valkyrie::Types::ID)
```

Comment:
It looks as if Accessions are setup here to have Coin members. Is this the relationship we want? We discussed in Slack inserting the `accession_number` into a property on the Coin resource to make the connection. Also, using member_ids this way means we end up with Coins with two parents; an Issue and an Accession.